src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-form/iscsi-target-form.component.html

```diff
@@ -8,7 +8,8 @@
     <div class="panel panel-default">
       <div class="panel-heading">
         <h3 class="panel-title"
-            i18n>{isEdit, select, 1 {Edit} other {Add}} target</h3>
+            i18n="form title|Example: Create Pool@@formTitle"
+            >{{ action | titlecase }} {{ resource | titlecase }}</h3>
```

Comment:
We decided not to use the `>` in a separate line from the attrs.

Reply:
I've used now the attribute alignment, but given the weirdness we're dealing with (i18n and whitespacing), I think the trailing `>` is a minor issue :stuck_out_tongue: